src/mono/wasm/build/WasmApp.Common.targets

```diff
@@ -173,8 +173,7 @@
     <EnableDefaultWasmAssembliesToBundle Condition="'$(EnableDefaultWasmAssembliesToBundle)' == ''">true</EnableDefaultWasmAssembliesToBundle>
     <!-- VS uses DeployOnBuild, and sdk sets _IsPublishing -->
     <WasmBuildOnlyAfterPublish Condition="'$(WasmBuildOnlyAfterPublish)' == '' and ('$(DeployOnBuild)' == 'true' or '$(_IsPublishing)' == 'true')">true</WasmBuildOnlyAfterPublish>
-    <WasmGenerateAppBundle Condition="'$(WasmGenerateAppBundle)' == '' and '$(OutputType)' != 'Library'">true</WasmGenerateAppBundle>
-    <WasmGenerateAppBundle Condition="'$(WasmGenerateAppBundle)' == ''">false</WasmGenerateAppBundle>
+    <WasmGenerateAppBundle Condition="'$(WasmGenerateAppBundle)' == ''">true</WasmGenerateAppBundle>
```

Comment:
This needs a condition as well, doesn't it? (so that the target doesn't run until it's a wasm library)

Reply:
shouldn't it run even when exe ?

Reply:
Yeah, but until RID=wasm+Output type=exe meant do wasm build, which we need to leave intact until wasmconsole template and runtime tests migrate to Wasm SDK. For library mode we also need to check for Wasm SDK use. (That's my theory, maybe something else guards this)